sccp/figures/common.py

```diff
@@ -17,6 +17,10 @@
 import os
 from os.path import join
 from pandas.plotting import parallel_coordinates as pc
+from parafac2 import parafac2_nd
+from sklearn.linear_model import LogisticRegressionCV, LogisticRegression
+from sklearn.metrics import roc_auc_score, RocCurveDisplay
```

Comment:
need to be able to run Pf2 for different ranks in `testPf2Ranks`; need to be able to do cross validated regression/fit after, and calculate ROC AUC